controllers/nmstate_controller.go

```diff
@@ -83,6 +89,12 @@ func (r *NMStateReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, err
 	}
 
+	// We only want one instance of NMState. Ignore anything after that.
+	if len(instanceList.Items) > 0 && instanceList.Items[0].Name != req.Name {
+		r.Log.Info("Ignoring NMState.nmstate.io because one already exists")
+		return ctrl.Result{}, nil
+	}
```

Comment:
We don't want to delete the new created one ?

Reply:
That's a good idea. I'll add, I mean... delete that. :)